File: fun/cuisine-chooser.py
"""
Using random to generate list
"""

import random

cuisine_type = [
    "Indian",
    "Salvadoran",
    "Mediterranean",
    "Deli",
    "Mexican",
    "Chinese",
    "Thai",
    "Italian",
    "BBQ",
    "Japanese",
    "Korean",
    "Greek",
    "Food Hall",
]
output_type = random.sample(cuisine_type, k=3)
MESSAGE = "Your cuisine choice is: "

print(MESSAGE + ", ".join(output_type))

File: fun/cuisine-chooser.R
# List of cuisine types
cuisine_type <- c("Indian", "Salvadoran", "Mediterranean", "Deli", 
                  "Mexican", "Chinese", "Thai", "Italian", "BBQ", 
                  "Japanese", "Korean", "Greek", "Food Hall")

# Generate random sample of 3 cuisines
output_type <- sample(cuisine_type, size = 3)

# Message
message <- "Your cuisine choice is: "

# Combine message and list elements with comma separation
result <- paste(message, paste(output_type, collapse = ", "), sep = "")

# Print the result
cat(result, "\n")
